bw2io/backup.py

```diff
@@ -13,20 +13,23 @@
 
 
 def backup_data_directory(
-    timestamp: Optional[bool] = True, dir_backup: Optional[Union[str, Path]] = None
```

Comment:
Don't love changing argument order, but we haven't released with this yet and the new order is more logical, so OK.